### PR TITLE
New fonts

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### Reference Issues/PRs
#60 #59 
#### What does this implement/fix? Explain your changes.
Add 5 new fonts :
1. fancy85
2. fancy86
3. contouring3
4. contouring4
5. love2

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +18,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 	9. fancy82
 	10. fancy83
 	11. fancy84
+	12. contouring3
+	13. contouring4
+	14. love2
+- `get_font_dic` function
```
Comment (sadrasabouri) [inline CHANGELOG.md]: Thank you for this :handshake:

File: otherfile/font_check.py [hunk 2]
```diff
@@ -68,10 +68,12 @@ def print_result(flag_list, message_list):
     for font1 in Font_List:
         for font2 in Font_List:
             if Font_List.index(font1) < Font_List.index(font2):
-                if len(art.get_font_dic(font1)) == len(art.get_font_dic(font2)):
+                if len(
```
Comment (sadrasabouri) [inline otherfile/font_check.py]: What is this for?

Comment (sepandhaghighi) [reply otherfile/font_check.py]: @sadrasabouri pep8 rule about max length of lines.

Comment (sadrasabouri) [reply otherfile/font_check.py]: Oh, Thanks.

**Pull request merged**